conda_recipe/pytorch_deps_environment.yml

```diff
@@ -0,0 +1,22 @@
+# This file may be used to create an environment using:
+# $ conda env create --name <env> --file conda_recipe/environment.yml
```

Comment:
probably should be `... conda_recipe/pytorch_deps_environment.yml`

Reply:
Missed this, should be correct now.